PS8lite.Rmd

```diff
@@ -202,8 +204,28 @@ Using `model_rf` make predictions and submit to Kaggle.
 
 ```{r}
 
+#Using the model_rf to make predictions
+test <- test %>% 
+  mutate(
+    logSalePrice_hat_rf = predict(model_rf, test),
+    SalePrice_hat_rf= exp(logSalePrice_hat_rf) -1)
+glimpse(test)
+```
+
+
+
+
+
+
+```{r}
+
+#Submission to Kaggle 
+subRF <- test %>%
+  mutate(SalePrice = SalePrice_hat_rf) %>%
+  select(Id, SalePrice)
+write_csv(subRF, "data/subRF.csv")
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.